Utilities/StaticAnalyzers/src/ClangCmsCheckerPluginRegister.cpp

```diff
@@ -87,7 +92,7 @@ extern "C" void clang_registerCheckers(clang::ento::CheckerRegistry &registry) {
   registry.addChecker<clangcms::ThrUnsafeFCallChecker>(
       "threadsafety.ThrUnsafeFCallChecker", "Reports calls of known thread unsafe functions", "no docs");
   registry.addChecker<clangcms::ESRGetChecker>(
-      "deprecated.ESRecordGetChecker", "Checks for calls to EventSetupRecord::get", "no docs");
+      "cms.CodeRules.ESRecordGetChecker", "Checks for calls to EventSetupRecord::get", "no docs");
```

Comment:
I think that @makortel [commented](https://github.com/cms-sw/cmssw/pull/36424#issuecomment-989171826) to move under "CodeRules" the newly added checks, not this already existing one...

Reply:
Thanks @perrotta, I was too quick in signing (removed the `+1` now). It is indeed the added `deprecated.psetExistsChecker` that should become `cms.CodeRules.psetExistsChecker`.

Reply:
Good catch!